bats/tests/registry/creds.bats

```diff
@@ -103,6 +108,22 @@ EOF
     wait_for_container_engine
 }
 
+@test 'verify credential is set correctly' {
+    verify_default_credStore
+}
+
+verify_default_credStore() {
+    local CREDHELPER_NAME="$(basename "$CRED_HELPER" .ext | sed s/docker-credential-//)"
```

Comment:
I assume this should strip the Windows `.exe` extension:

```suggestion
    local CREDHELPER_NAME="$(basename "$CRED_HELPER" .exe | sed s/^docker-credential-//)"
```

It doesn't really matter, but maybe also anchor the regex at the beginning of the name.

Reply:
Yes on the `ext/exe` mixup -- I was thinking "strip the extension", and mis-abbreviated it.